lib/google_crawler_web/error_handler.ex

```diff
@@ -24,4 +24,18 @@ defmodule GoogleCrawlerWeb.ErrorHandler do
     conn
     |> handle(:forbidden)
   end
+
+  def full_message(changeset) do
+    Ecto.Changeset.traverse_errors(changeset, fn {msg, opts} ->
+      Enum.reduce(opts, msg, fn {key, value}, acc ->
+        String.replace(acc, "%{#{key}}", to_string(value))
+      end)
+    end)
+    |> Enum.map(fn {attribute, message} ->
+      formatted_attribute = Recase.to_sentence(Atom.to_string(attribute))
```

Comment:
Is there a reason why we're using `Recase` over [Phoenix.Naming.humanize()](https://hexdocs.pm/phoenix/Phoenix.Naming.html#humanize/1)?
since `attribute` is an atom already. 🤔

Reply:
Good idea, I will update to `humanize/1` 👍